parquet-common/src/main/java/org/apache/parquet/bytes/BytesInput.java

```diff
@@ -376,7 +378,15 @@ void writeInto(ByteBuffer buffer) {
         ByteBuffer workBuf = buffer.duplicate();
         int pos = buffer.position();
         workBuf.limit(pos + byteCount);
-        Channels.newChannel(in).read(workBuf);
```

Comment:
Is there any other place that is used like this?

Reply:
I went through the [original PR](https://github.com/apache/parquet-java/pull/1278) and found nothing else, it would be great if others have a double check.